zrml/global-disputes/src/global_disputes_pallet_api.rs

```diff
@@ -35,7 +35,7 @@ pub trait GlobalDisputesPalletApi<MarketId, AccountId, Balance> {
     ///
     /// Returns the dispute mechanism's report if available, otherwise `None`. If `None` is
```

Comment:
The comment about return value seems outdated.

Reply:
6b2f233ae7fe3012951c880b66b718f7f7c7dea9